test/others/criu-ns/Makefile

```diff
@@ -0,0 +1,2 @@
+run:
```

Comment:
```Suggestion
run:
	@make -C ../.. zdtm_ct
```

Reply:
is this similar to `../../zdtm_ct run.py`

Reply:
No. It runs `make zdtm_ct` in the `test` folder to ensure that `zdtm_ct` would be compiled.
```
$ sudo git clean -dfx
$ make -j8
$ make -C test/others/criu-ns/ run
make: Entering directory '/home/radostin/Projects/coordinated-checkpointing/criu/test/others/criu-ns'
../../zdtm_ct run.py
make: ../../zdtm_ct: No such file or directory
make: *** [Makefile:2: run] Error 127
```

Reply:
No. It runs `make zdtm_ct` in the `test` folder to ensure that `zdtm_ct` would be compiled.
```
$ sudo git clean -dfx
$ make -j8
$ make -C test/others/criu-ns/ run
make: Entering directory '/criu/test/others/criu-ns'
../../zdtm_ct run.py
make: ../../zdtm_ct: No such file or directory
make: *** [Makefile:2: run] Error 127
```

Reply:
understood, thanks!

Reply:
done